### PR TITLE
golint fixes in pkg/kubectl/cmd/wait

### PR DESCRIPTION
**Pull request by sayanarijit**
**What this PR does / why we need it**:

Refactoring #68176 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/kubectl/cmd/wait/wait.go [hunk 1]
```diff
@@ -67,10 +67,10 @@ var (
 // errNoMatchingResources is returned when there is no resources matching a query.
 var errNoMatchingResources = errors.New("no matching resources found")
 
-// WaitFlags directly reflect the information that CLI is gathering via flags.  They will be converted to Options, which
+// Flags directly reflect the information that CLI is gathering via flags.  They will be converted to Flags, which
 // reflect the runtime requirements for the command.  This structure reduces the transformation to wiring and makes
```
Comment (stewart-yu) [inline pkg/kubectl/cmd/wait/wait.go]: Why change `WaitFlags ` to `Flags`?

File: pkg/kubectl/cmd/wait/wait.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ import (
 )
 
 var (
-	wait_long = templates.LongDesc(`
+	waitLong = templates.LongDesc(`
 		Experimental: Wait for a specific condition on one or many resources.
```
Comment (stewart-yu) [inline pkg/kubectl/cmd/wait/wait.go]: The same

File: pkg/kubectl/cmd/wait/wait.go (hunk 1 above)
Comment (stewart-yu) [reply pkg/kubectl/cmd/wait/wait.go]: emmm......, can you paster the error in here?

File: pkg/kubectl/cmd/wait/wait.go (hunk 2 above)
Comment (stewart-yu) [reply pkg/kubectl/cmd/wait/wait.go]: Great, seems reasonable

File: pkg/kubectl/cmd/wait/wait.go (hunk 1 above)
Comment (stewart-yu) [reply pkg/kubectl/cmd/wait/wait.go]: IMO, Check where transfer the `wait.WaitFlags`, and rename the import file

Comment (stewart-yu) [reply pkg/kubectl/cmd/wait/wait.go]: IMO, named like that seems too easy, How about `CLIWaitFlags`?

File: pkg/kubectl/cmd/wait/wait.go [hunk 3]
```diff
@@ -184,17 +184,19 @@ func conditionFuncFor(condition string) (ConditionFunc, error) {
...
 }
 
+// UIDMap maps ResourceLocation with UID
 type UIDMap map[ResourceLocation]types.UID
 
-// WaitOptions is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
+// Options is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
 // command, making the logic itself easy to unit test with our existing mocks.
-type WaitOptions struct {
+type Options struct {
```
Comment (stewart-yu) [inline pkg/kubectl/cmd/wait/wait.go]: How about `CLIWaitOptions`?

Comment (sayanarijit) [reply pkg/kubectl/cmd/wait/wait.go]: Not sure. need more opinions here... I used naming convention as followed [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubectl/apply/strategy/merge.go#L22) and [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubelet/server/remotecommand/httpstream.go#L42)

File: pkg/kubectl/cmd/wait/wait.go (hunk 1 above)
Comment (sayanarijit) [reply pkg/kubectl/cmd/wait/wait.go]: Not sure. need more opinions here... I used naming convention as followed [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubectl/apply/strategy/merge.go#L22) and [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubelet/server/remotecommand/httpstream.go#L42)

**Comment by sayanarijit** [comment]
Closing this I will not be able to follow up on this for some time.

**Pull request closed without merging**